docs/home/howto/LOGGING.md

```diff
@@ -1,119 +1,124 @@
 # Centralized logging setup
 
-For centralized logging Epiphany uses [OpenDistro for Elasticsearch](https://opendistro.github.io/for-elasticsearch/).
-In order to enable centralized logging, be sure that `count` property for `logging` feature is greater than 0 in your
+For centralized logging Epiphany uses [OpenSearch](https://opensearch.org/) stack - an opensource successor<sup>[1]</sup> of Elasticsearch & Kibana projects.
```

Comment:
Is it really a successor? Open source should be written separately.

Reply:
ok
Implemented.